crowbar_framework/lib/crowbar/lock/shared_non_blocking.rb

```diff
@@ -16,6 +16,7 @@
 #
 
 module Crowbar
+  include RemoteNode
```

Comment:
I would add a comment to explain why this is needed, so people don't remove it blindly.